src/github.com/matrix-org/dendrite/roomserver/roomserver-integration-tests/main.go

```diff
@@ -17,8 +18,10 @@ var (
 	zookeeperURI = defaulting(os.Getenv("ZOOKEEPER_URI"), "localhost:2181")
 	// The URI the kafka server is listening on.
 	kafkaURI = defaulting(os.Getenv("KAFKA_URIS"), "localhost:9092")
+	// The address the roomserver should listen on.
+	roomserverAddr = defaulting(os.Getenv("ROOMSERVER_URI"), "localhost:9876")
 	// How long to wait for the roomserver to write the expected output messages.
-	timeoutString = defaulting(os.Getenv("TIMEOUT"), "10s")
+	timeoutString = defaulting(os.Getenv("TIMEOUT"), "60s")
```

Comment:
Why the increase? This is all localhost right, so I hope it doesn't take >10s...